CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 3.1.1 (2019-06-10)
+
+- Fixed the flyout component to handle click events that originate within its contents. [#44](https://github.com/blackbaud/skyux-flyout/pull/44)
+- Fixed the flyout component to style the resize handle and iterator buttons correctly in Internet Explorer. [#45](https://github.com/blackbaud/skyux-flyout/pull/45)
+- Fixed the flyout component to only resize to twenty pixels less than the window width. [#46](https://github.com/blackbaud/skyux-flyout/pull/46)
```

Comment:
How about something like "Fixed the flyout component to never resize larger than twenty pixels less than the window?" Docs, feel free to chime in here...

Reply:
How about "Fixed the flyout component to limit its maximum width to 20 pixels less than the window's width."

Reply:
That's fine